### PR TITLE
Update the list of libraries

### PR DESCRIPTION
**Pull request by ivacf**

### CODE REVIEW

File: android_libraries.md [hunk 1]
```diff
@@ -1,25 +1,21 @@
...
+ - [Espresso](https://code.google.com/p/android-test-kit/wiki/Espresso)
  - [Robolectric](http://robolectric.org/)
+ - [Mockito](https://github.com/mockito/mockito)
+ - JUnit 
 
 ## Standard Libraries
 
 ### Architecture
-- [Dagger](http://square.github.io/dagger/)
+- [RxJava](https://github.com/ReactiveX/RxJava) & [RxAndroid](https://github.com/ReactiveX/RxAndroid)
```
Comment (jsonfry) [inline android_libraries.md]: yay

File: android_libraries.md [hunk 2]
```diff
@@ -1,25 +1,21 @@
...
 - [Otto](http://square.github.io/otto/)
-- [Pig](http://github.com/ribot/pig)
```
Comment (jsonfry) [inline android_libraries.md]: good

File: android_libraries.md [hunk 3]
```diff
@@ -1,25 +1,21 @@
...
-
 
 ### Networking
 - [Okhttp](http://square.github.io/okhttp/)
-- [Volley](https://android.googlesource.com/platform/frameworks/volley/)
+- [Retrofit](https://github.com/square/retrofit)
```
Comment (jsonfry) [inline android_libraries.md]: even better

File: android_libraries.md [hunk 4]
```diff
@@ -27,8 +23,6 @@
 - [Butterknife](http://jakewharton.github.io/butterknife/)
 - [Easy Adapter](https://github.com/ribot/easy-adapter)
 - [Calligraphy](https://github.com/chrisjenx/Calligraphy)
-
-### Other
-- [Auto Parcel](https://github.com/frankiesardo/auto-parcel)
```
Comment (jsonfry) [inline android_libraries.md]: Did we ever use this?

Comment (ivacf) [inline android_libraries.md]: We tried it but it wasn't very good so we never used it on a real project

Comment (jsonfry) [inline android_libraries.md]: good :)

**Pull request merged**